react/features/chat/components/web/NewMessagesButton.tsx

```diff
@@ -0,0 +1,90 @@
+import { makeStyles } from '@material-ui/core';
+import React, { MouseEventHandler } from 'react';
+import { WithTranslation } from 'react-i18next';
+
+import { translate } from '../../../base/i18n/functions';
+import Icon from '../../../base/icons/components/Icon';
+import { IconArrowDown } from '../../../base/icons/svg/index';
+import { withPixelLineHeight } from '../../../base/styles/functions.web';
+export interface INewMessagesButtonProps extends WithTranslation {
+    inputChatHeight: number;
+    onGoToFirstUnreadMessage: MouseEventHandler;
+}
+
+const useStyles = makeStyles((theme: any) => {
+    return {
+        container: {
+            position: 'absolute',
+            left: 'calc(50% - 72px)',
+            bottom: '15px'
+        },
+
+        newMessagesButton: {
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            width: '143px',
```

Comment:
Do we need to have a fixed width? Considering the content will always be the same I think it should be fine without it. Please test this